src/version.h.in

```diff
@@ -0,0 +1,11 @@
+// Get version number from CMAKE
+#ifndef INCLUDE_GUARD
```

Comment:
Can you swap this to `#pragma once` please? That's what we've used elsewhere in riesling

Reply:
Yes of course!